.github/release/release_notes.md

```diff
@@ -0,0 +1,85 @@
+other Scala CLI sub-commands.
+
+This feature has been added in preparation for the `publish` command, stay tuned for future announcements.
+
+Added in [#1056](https://github.com/VirtusLab/scala-cli/pull/1056) by @alexarchambault
+
+## Prioritise non-script main classes
+
+When trying to run a directory containing scripts and just a single non-script main class, the non-script main class
+will now be prioritised and run by default.
+
+```
+$ scala-cli .
+Running Hello. Also detected script main classes: scripts.AnotherScript_sc, scripts.Script_sc
+You can run any one of them by passing option --main-class, i.e. --main-class scripts.AnotherScript_sc
+All available main classes can always be listed by passing option --list-main-classes
+Hello world
+```
+
+Changed in [#1095](https://github.com/VirtusLab/scala-cli/pull/1095) by @Gedochao
+
+## Important bugfixes
+
+### Enable downloading Scala versions outside the Coursier cache
+
+Scala CLI is now able to download the newest Scala versions, even if they're not yet available in the Coursier cache. In
+case of a cache miss, it force-retries and fetches the version, if at all possible.
+This addresses [#1090](https://github.com/VirtusLab/scala-cli/issues/1090)
+
+Fixed in [#1096](https://github.com/VirtusLab/scala-cli/pull/1096) by @lwronski
+
+### Bloop now uses `JAVA_HOME` by default
+
+Bloop should now pick up on the JDK available in `JAVA_HOME`. It was formerly necessary to pass `--bloop-jvm system`
+explicitly. This addresses [#1102](https://github.com/VirtusLab/scala-cli/issues/1102)
+
+Fixed in [#1084](https://github.com/VirtusLab/scala-cli/pull/1084) by @lwronski
+
+### The `-coverage-out` option now accepts relative paths
+
+Scala CLI now correctly processes relative paths when passed to the `-coverage-out` option. Formerly,
+the `scoverage.coverage` file would not be properly generated when a relative path was passed.
+This addresses [#1072](https://github.com/VirtusLab/scala-cli/issues/1072)
+
+Fixed in [#1080](https://github.com/VirtusLab/scala-cli/pull/1080) by @lwronski
+
+## Other changes
+
+### Documentation PRs
+
+* Improve scripts guide by @Gedochao in [#1074](https://github.com/VirtusLab/scala-cli/pull/1074)
+* Update installation instructions for Nix by @kubukoz in [#1082](https://github.com/VirtusLab/scala-cli/pull/1082)
+* Tweak docs by @alexarchambault in [#1085](https://github.com/VirtusLab/scala-cli/pull/1085)
+* Some typos & rewording on the single-module projects use case page by @Baccata
+  in [#1089](https://github.com/VirtusLab/scala-cli/pull/1089)
+
+### Fixes
+
+* Add suffix to project name which contains virtual files by @lwronski
+  in [#1070](https://github.com/VirtusLab/scala-cli/pull/1070)
+
+### Build and internal changes
+
+* Update scala-cli.sh launcher for 0.1.7 by @github-actions in [#1076](https://github.com/VirtusLab/scala-cli/pull/1076)
+* Add config command by @alexarchambault in [#1056](https://github.com/VirtusLab/scala-cli/pull/1056)
```

Comment:
Do we need this line, as there's a section about it above?
```suggestion
```

Reply:
Ah yeah, indeed we don't